resolve depencency conflicts

kenohassler opened the pull request:
The current version does not build, presumably because dependabot created some dependency conflicts.
Resolved by deleting requirements_lock.txt and letting it regenerate with `bazel run //:requirements.update`.

kenohassler commented:
I'll add two more commits, thanks for the feedback :)

kenohassler commented on wgkex/worker/netlink_test.py:
```diff
@@ -1,24 +1,21 @@
+from pyroute2 import IPRoute, WireGuard  # noqa: E402
 
-from wgkex.worker import netlink
+from wgkex.worker import netlink  # noqa: E402
```
I had to silence these two issues because actually the imports *cannot* be re-sorted – they need to be below the mocking code.

grische commented:
Is there an alternative for `--experimental_cc_coverage` in Bazel 8? Are we not using the output?

T0biii commented:
https://github.com/bazelbuild/bazel/pull/7556 (Rename --experimental_cc_coverage to --incompatible_cc_coverage.)
https://github.com/bazelbuild/bazel/pull/7933 (Remove --incompatible_cc_coverage)
The Paramter was still accepted but with a `Obsolete, no effect` Info
so it was already removed in 2019 bevor this Project even started lol (could be that we where using a older version at this time?).
and finally removed in 2024
https://github.com/bazelbuild/bazel/pull/23375 (Remove 6 no-op CLI flags)


one thing we need to look at is this info from the CI:
`There were tests whose specified size is too big. Use the --test_verbose_timeout_warnings command line option to see which ones these are.
`
https://github.com/freifunkMUC/wgkex/actions/runs/17741785660/job/50418089722?pr=190#step:5:874

kenohassler commented:
I added a commit that adds the recommended `--test_verbose_timeout_warnings` option, so we should see which tests run too long in the CI output.
Also I replaced `--javabase=@bazel_tools//tools/jdk:remote_jdk11` with `--java_runtime_version=remotejdk_11`, as there were warnings about this option being deprecated. The alternative is [documented here](https://bazel.build/concepts/platforms#java).

T0biii commented:
@kenohassler i think you can remove `--test_verbose_timeout_warnings` again.
Then this should be reday to merge.

if you want you can add the size to the tests that sould "fix" the problem but this can also be done in a sperate PR if you dont want to

kenohassler commented:
I added the "fix", now that we already discussed it might as well have it 😄
(I would keep `--test_verbose_timeout_warnings` because it doesn't generate any extra output if there are no warnings, but can remove it if you prefer. Otherwise, ready to merge.)
Thanks for the thorough review!

grische commented:
Thanks a lot for all the changes! This is a nice and thorough cleanup.
